scripts/release.sh

```diff
@@ -54,6 +64,23 @@ if ( echo "$version" | grep -q "SNAPSHOT" ); then
     exit 1
 fi
 
+if [ -z "${DRYRUN:-}" ]; then
+    gradle_build_args="--no-build-cache --warning-mode all --refresh-dependencies clean build publishToMavenLocal"
```

Comment:
`build` takes quite a long time to run all the tests. When I want to run `japicmp.sh`, I usually use `assemble` task. It's enough to compile and generate jars.
`publishToMavenLocal` is not required, japicmp.sh will take artifacts from the `build` directory

Reply:
`publishToMavenLocal` was done bcz some of the related release tasks (for related projects) involve validating with local builds. I can remove though.